apps/api/migrations/encrypt-credentials/encrypt-credentials-migration.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { expect } from 'chai';
+import { encryptOldCredentialsMigration } from './encrypt-credentials-migration';
+import { UserSession } from '@novu/testing/src';
+import { ChannelTypeEnum } from '@novu/stateless';
+import { IntegrationRepository } from '@novu/dal/src';
+
+describe('Encrypt Old Credentials', function () {
+  let session: UserSession;
+  const integrationRepository = new IntegrationRepository();
+
+  beforeEach(async () => {
+    session = new UserSession();
+    await session.initialize();
+  });
+
+  it('should decrypt all old credentials', async function () {
+    await pruneIntegration(integrationRepository);
+
+    const data = {
+      providerId: 'sendgrid',
+      channel: ChannelTypeEnum.EMAIL,
+      active: false,
+    };
+
+    for (let i = 0; i < 2; i++) {
+      await integrationRepository.create({
+        _environmentId: session.environment._id,
+        _organizationId: session.organization._id,
+        providerId: data.providerId + i,
+        channel: data.channel,
+        credentials: { user: 'smith' + i, apiKey: '123' + i, secretKey: 'abc' + i },
+        active: data.active,
+      });
+    }
+
+    const newIntegration = await integrationRepository.find({});
+
+    expect(newIntegration.length).to.equal(2);
+
+    await encryptOldCredentialsMigration();
+
+    const encryptIntegration = await integrationRepository.find({});
+
+    for (const integrationKey in encryptIntegration) {
+      const integration = encryptIntegration[integrationKey];
+
+      expect(integration.credentials.user).to.equal('smith' + integrationKey);
+      expect(integration.credentials.apiKey).to.contains('nvsk.');
+      expect(integration.credentials.secretKey).to.contains('nvsk.');
+    }
```

Comment:
Could you add an assertion for encryptIntegration? In cases it's empty the test will always pass :)

Reply:
Hey @scopsy could you elaborate a bit more not sure I followed